Dockerfile

```
@@ -35,6 +35,7 @@ ENV \
 	AVAILABLE_HEALTH_CHECK=true\
 	AVAILABLE_HEALTH_CHECK_GRACE_PERIOD=10\
 	AVAILABLE_HEALTH_CHECK_THRESHOLD=0\
+	AVAILABLE_HEALTH_CHECK_IGNORED_ERRORS=Unhealthy,FailedScheduling\
```

`Unhealthy`?
Also, what are the ramifications for fastrollbacks? If we get this error during a deployment of a fastrollback, and kit ignores it, the service's selector will never get updated correctly?

---

That was pre-existing, I didn't add that so it shouldn't change the current behavior.  We'd still eventually time out the deployment of it's never successful

---

Right, but if kit ignores there error, how would it know to switch the service selector after its successful? Will kit keep watching after ignoring this error?

---

So there's another [process](https://github.com/InVisionApp/kit-deployer/blob/master/src/lib/kubectl.js#L20) that's setup on an interval that's emitting new events from it's watching Kubernetes, this return prevents us from treating these events as errors by returning before [this error handling block](https://github.com/InVisionApp/kit-deployer/blob/master/src/lib/health-check.js#L102-L148).  So instead of entering a timeout and checking if the number of errors has reached it's threshold, we continue processing new events.

If we're unable to complete the deploy before the deploy's timeout kicks in, we'd just fail the deploy so the service wouldn't be switched.  If we end up clearing our FailureToFit errors, then we'd simple see the success of the deploy and move forward with service selector switchover.

---

ok, perfect. Thanks for checking :)

---

:wave: :smile: